app/assets/javascripts/old_samfundet/interviews.js

```diff
@@ -15,9 +15,22 @@ $(function () {
       type: 'text'
   });
 
+  $.tablesorter.addParser({
```

Comment:
Given that we're still modifying this file, should we just move it outside to `javascripts/applicants/interviews.js`?

Reply:
Yeah, makes sense.